.gitignore

```diff
@@ -20,6 +20,7 @@
 *.code-workspace
 
 .DS_Store
+.cache
```

Comment:
This `.cache` is generated by VS Code plugin for C++ I am using - I hope it is not a problem :)

Reply:
Yes, seems reasonable.